### ISSUE TITLE
Should someone else maintain this?

### ISSUE THREAD

**Issue by junaid1460** [Yelp/elastalert#2946]

**Comment by Qmando** [comment]
Yes. That will happen in the future. Can't give you an exact timeline.

**Comment by nickbabkin** [comment]
@Qmando please don't let this die without support.
Usecase for ElastAlert is pretty unique and it stands well in the ELK ecosystem as an open source alternative to licensed Elastic alerting features. Moreover, the functionality is way way richer than what Elastic guys have under the hood.

You did a great job with building maintaining it, now please hand it over to someone who deserves.


Cheers!

**Comment by fzyzcjy** [comment]
Hi I wonder whether there are some updates? This is a very helpful and useful project!

**Comment by mrfroggg** [comment]
People can follow on this discussion here to pitch in their opinions about an official fork and the process of it: [jertel#77](https://github.com/jertel/elastalert/discussions/77)
